src/browser/BrowserShared.cpp

```diff
@@ -31,6 +31,9 @@ namespace BrowserShared
         const auto serverName = QStringLiteral("/org.keepassxc.KeePassXC.BrowserServer");
 #if defined(KEEPASSXC_DIST_SNAP)
         return QProcessEnvironment::systemEnvironment().value("SNAP_USER_COMMON") + serverName;
+#elif defined(KEEPASSXC_DIST_FLATPAK)
+        return QStandardPaths::writableLocation(QStandardPaths::RuntimeLocation)
+            + "/app/" + "org.keepassxc.KeePassXC" + serverName;
```

Comment:
@droidmonkey we might want to reuse the `applicationIconName` function here too.

Reply:
Maybe it's a bit overkill. There's other instances of the fully qualified name in other places. Let's leave that refactoring for later.